beluga_amcl/docs/PARAMETERS.md

```diff
@@ -1,8 +1,10 @@
 # Beluga AMCL Parameter Reference
 
-This page describes the parameters supported by Beluga AMCL including a comparison table with Nav2 AMCL and compatibility notes.
+This page describes the parameters supported by Beluga AMCL including comparison tables with Nav2 AMCL and AMCL, along with relevant compatibility notes.
 
-## ROS Interface Parameters
+## ROS 2
```

Comment:
nit: A table of contents in this page would be super helpful now that we have this many sections. Alternatively, we could have two separate pages per major ROS version.

Reply:
Added a TOC in https://github.com/Ekumen-OS/beluga/pull/218/commits/2c6130aad012ca003cf1af8e863910f6d2a21ff0.